config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -68,14 +68,15 @@ spec:
                 pytorch:
                   properties:
                     modelClassName:
-                      description: Name of the model class for PyTorch model
+                      description: Defaults PyTorch model class name to 'PyTorchModel'
```

Comment:
We support both, taking it from a user, and if not specified, use a default. Is the description sufficient?